.github/workflows/deadnix.yaml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [ main ]
     tags: [ '*' ]
-  pull_request:
```

Comment:
Why?

Reply:
It failed previously: https://github.com/NuschtOS/dn42.nix/actions/runs/12108631763/job/33756971217?pr=50#step:5:99

Only seems to work on the same repo?

Reply:
I think the idea was that it PRs your repo with the fix but that doesn't seem to work as intended.